mkdocs/config/config_options.py

```diff
@@ -647,6 +676,13 @@ def run_validation(self, value):
                     stack.insert(0, frame)
                 tb = ''.join(traceback.format_list(stack))
 
+                if markdown_version >= "3.4":
```

Comment:
The value of `markdown_version` can be `None` which is not comparable.

It could also be these:
```python
"3.10" <= "3.4"
"10.0" <= "3.4"
```
-which is not correct